python/lsst/ip/diffim/getTemplate.py

```diff
@@ -277,25 +278,33 @@ def run(self, *, coaddExposureHandles, bbox, wcs, dataIds, physical_filter):
         for tract in coaddExposureHandles:
             maskedImages, catalog, totalBox = self._makeExposureCatalog(coaddExposureHandles[tract],
                                                                         dataIds[tract])
+            warpedBox = computeWarpedBBox(catalog[0].wcs, bbox, wcs)
+            warpedBox.grow(5)  # to ensure we catch all relevant input pixels
```

Comment:
My understanding of `computeWarpedBBox` is that that already errs on the side of a larger box, so growing it here is probably unnecessary. That said, an extra 5 pixels shouldn't make much of a difference, and it will still be fewer pixels than before.

Reply:
I found that without growing the box by a few pixels, the corners in the tests weren't correct, so I decided to be a bit generous here.